glide.yaml

```diff
@@ -10,7 +10,7 @@ excludeDirs:
 - toolbox
 import:
 - package: github.com/coreos/etcd
-  version: release-3.1
+  version: 20490caaf0dcd96bb4a95e40625559def8ef5b04
```

Comment:
@jbw976 This hash actually corresponds to v3.1.5, which is newer than what we were using.